Coverage/CoverageRunner.h

```diff
@@ -413,12 +413,12 @@ struct CoverageRunner
 		ZeroMemory(&pi, sizeof(pi));
 
 		LPSTR args = NULL;
-		if (options.ExecutableArguments.size() > 0)
+		if( !options.ExecutableArguments.empty() )
 		{
 			args = &*options.ExecutableArguments.begin();
 		}
 
-		auto result = CreateProcess(options.Executable.c_str(), args, NULL, NULL, FALSE, DEBUG_PROCESS, NULL, NULL, &si, &pi);
+		auto result = CreateProcess(NULL, args, NULL, NULL, FALSE, DEBUG_PROCESS, NULL, options.WorkingDirectory.c_str(), &si, &pi);
```

Comment:
Interesting... are you sure this is correct? I haven't tested it, but I would expect removing the executable path gives problems with argv[0]